.storybook/preview.tsx

```diff
@@ -0,0 +1,37 @@
+import '@ubie/design-tokens/dist/tokens.css';
+import { Preview } from '@storybook/react';
+// @ts-ignore
+import { resetCss, ResetCssDecorator } from './decorators/ResetCssDecorator';
+
+export const parameters = {
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+};
+export const tags = ['autodocs'];
+
+const preview: Preview = {
```

Comment:
The lack of Reset CSS was causing the appearance of the actual product to change.
However, the reset CSS may differ depending on the productbeing used.
So we have implemented a system that allows you to switch between reset CSS.
First, you can choose between “modern-css-reset” and none, and “modern-css-reset” is the default.

Reply:
https://github.com/user-attachments/assets/4c70c6cb-fe7c-4357-888b-62fe7e5cedfa